### PR TITLE
Replace ways to train content with new draft

### PR DESCRIPTION
**Pull request by peteryates**
### Trello card

https://trello.com/c/8HB2AjKg/1525-content-ways-to-train-redevelop-page-to-focus-more-on-user-considerations

### Context and changes

This is a direct import of @emmafrith's [work](https://docs.google.com/document/d/1xwsqWValXzc61xPaFrXHIL1-DNuxrAvZdK3zAnRjj38/edit#heading=h.ela620qkaqh2).

### CODE REVIEW

**Comment by peteryates** [comment]
Moving out of draft in an attempt to get it to build a review app...

File: app/views/content/ways-to-train.md [hunk 1]
```diff
@@ -42,208 +42,151 @@ keywords:
...
 
-$school-direct-fee-funded$
+Courses typically take one year but you may be able to study part-time.
 
-You’ll learn on the job while you work towards your Qualified Teacher Status (QTS).
+Fees are usually around £9,250 but you can:
 
-You’ll be based in a school but you will not be employed by them.
+* get [funding that you do not have to pay back](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#bursaries-and-scholarships) for certain subjects
+* apply for a [student loan](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#tuition-fee-and-maintenance-loans)
```
Comment (MylesJarvis) [inline app/views/content/ways-to-train.md:75]: These links need to be updated to '/funding-your-training#bursaries-and-scholarships' and '/funding-your-training#tuition-fee-and-maintenance-loans'

File: app/views/content/ways-to-train.md [hunk 2]
```diff
@@ -42,208 +42,151 @@ keywords:
...
+Despite the focus on training as you teach, many courses give you a PGCE or PGDE (an academic qualification) in addition to QTS.
 
-$school-direct-salaried$
+Courses typically take one year but you may be able to study part-time.
 
-You’ll be employed by a school as an unqualified teacher while you work towards your Qualified Teacher Status (QTS).
+Fees are usually around £9,250 but you can:
 
-You’ll earn a salary and do not need to pay any tuition fees.
+* get [funding that you do not have to pay back](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#bursaries-and-scholarships) for certain subjects
```
Comment (MylesJarvis) [inline app/views/content/ways-to-train.md:97]: These too

File: app/views/content/ways-to-train.md [hunk 3]
```diff
@@ -42,208 +42,151 @@ keywords:
...
+* apply for a [student loan](https://review-get-into-teaching-app-1182.london.cloudapps.digital/funding-your-training#tuition-fee-and-maintenance-loans)  
 
-This route typically requires 3 years of transferable work history.
+<div class="training-attributes">
+  <h4>School direct<br/>(fee-funded)</h4>
```
Comment (MylesJarvis) [inline app/views/content/ways-to-train.md:101]: @peteryates what's our general position on using <br> tags in the code? I've never seen any in before

**Comment by peteryates** [comment]
Replacing...

**Pull request closed without merging**